controller/work_item_comments.go

```diff
@@ -72,11 +73,11 @@ func (c *WorkItemCommentsController) Create(ctx *app.CreateWorkItemCommentsConte
 func (c *WorkItemCommentsController) List(ctx *app.ListWorkItemCommentsContext) error {
 	offset, limit := computePagingLimits(ctx.PageOffset, ctx.PageLimit)
 	return application.Transactional(c.db, func(appl application.Application) error {
-		_, err := appl.WorkItems().LoadByID(ctx, ctx.WiID)
+		wi, err := appl.WorkItems().LoadByID(ctx, ctx.WiID)
 		if err != nil {
 			return jsonapi.JSONErrorResponse(ctx, goa.ErrNotFound(err.Error()))
 		}
-		comments, tc, err := appl.Comments().List(ctx, ctx.WiID, &offset, &limit)
+		comments, tc, err := appl.Comments().List(ctx, wi.ID.String(), &offset, &limit)
```

Comment:
Will you change the interface of the `List` method to accept a `uuid.UUID` once?